casper/src/test/scala/coop/rchain/casper/util/rholang/Interactive.scala

```diff
@@ -75,7 +78,8 @@ class Interactive private (runtime: Runtime)(implicit scheduler: Scheduler) {
   def cleanUp(): Unit =
     runtime.close()
 
-  def checkpoint(name: String): Unit                  = checkpoints.update(name, runtime.space.createCheckpoint())
+  def checkpoint(name: String): Unit =
+    checkpoints.update(name, runtime.space.createCheckpoint().unsafeRunSync)
   def getCheckpoint(name: String): Option[Checkpoint] = checkpoints.get(name)
 
   def restore(name: String): Boolean =
```

Comment:
I think an `unsafeRunSync` is missing in the body of this function too. We call reset in here.